js/main.js

```diff
@@ -15,9 +16,115 @@ $form.addEventListener('submit', function (event) {
   data.profile.bio = $form.elements.bio.value;
   $form.reset();
   $image.setAttribute('src', 'images/placeholder-image-square.jpg');
+  viewSwapping('profile');
 });
 
 window.addEventListener('beforeunload', function (event) {
   var inputData = JSON.stringify(data);
   localStorage.setItem('profile', inputData);
 });
+
+function profileRender(data) {
+/* <div>
+    <div class="column-full">
+      <h1>data.profile.fullName</h1>
+    </div>
+    <div class="row">
+      <div class="column-half">
+        <img class="image-one" src=data.profile.avatarUrl alt="Avatar Image">
+      </div>
+      <div class="column-half">
+        <div class="flex">
+          <img class="image-two" src="https://encrypted-tbn0.gstatic.com/images?q=tbn%3AANd9GcTlx1OjBPG6fKXrDWNxQuIuCq5cI9qxT7goMA&usqp=CAU" alt="Profile icon">
+          <h4>data.profile.username</h4>
+        </div>
+        <div class="flex">
+          <img class="image-two" src="https://www.clipartkey.com/mpngs/m/179-1791285_little-location-icon.png" alt="Location icon">
+          <h4>data.profile.location</h4>
+        </div>
+        <p>data.profile.bio</p>
+      </div>
+    </div>
+  </div> */
+  var $container = document.createElement('div');
+  var $divFull = document.createElement('div');
+  $divFull.setAttribute('class', 'column-full');
+  var $h1 = document.createElement('h1');
+  $h1.textContent = data.profile.fullName;
+  $divFull.appendChild($h1);
+  $container.appendChild($divFull);
+
+  var $row = document.createElement('div');
+  $row.setAttribute('class', 'row');
+  var $divHalf = document.createElement('div');
+  $divHalf.setAttribute('class', 'column-half');
+  var $img1 = document.createElement('img');
+  $img1.setAttribute('class', 'image-one');
+  $img1.setAttribute('src', data.profile.avatarUrl);
+  $img1.setAttribute('alt', 'Avatar Image');
+  $divHalf.appendChild($img1);
+  $row.appendChild($divHalf);
+
+  var $divHalf2 = document.createElement('div');
+  $divHalf2.setAttribute('class', 'column-half');
+  var $flex = document.createElement('div');
+  $flex.setAttribute('class', 'flex');
+  var $img2 = document.createElement('img');
+  $img2.setAttribute('class', 'image-two');
+  $img2.setAttribute('src', 'https://encrypted-tbn0.gstatic.com/images?q=tbn%3AANd9GcTlx1OjBPG6fKXrDWNxQuIuCq5cI9qxT7goMA&usqp=CAU');
```

Comment:
While this will work for an icon, it is preferred that you make use of FontAwesome for your icons. It is even included in your html document for you already, take a look at the `head` element contents! :grin:

https://fontawesome.com/how-to-use/on-the-web/referencing-icons/basic-use